docs/developer/Supplemental_developer_guide.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Supplementary Developer Contribution Guidelines
+sidebar_position: 8
+---
+
+# Supplementary Developer Contribution Guidelines:
+
+1. Ensuring to squash similiar commits
```

Comment:
```suggestion
## Ensuring to squash similiar commits
```
How about replacing the numerical index with secondary headings?

Reply:
Modified as suggestion.